noxfile.py

```diff
@@ -249,7 +245,7 @@ def test(session, library):
     session.cd(library)
     if package:
         session.cd(f"packages/{package}")
-    unit(session)
+    default(session)
```

Comment:
Looking at the code above, `unit()` was just a thin wrapper around `defaults()`, so nothing changes functionally, correct? (And looking at the definition of `defaults()`, it's running the unit tests.)

Reply:
Correct